spec/paths/tracking@webhooks@{id}@resend.yaml

```
@@ -0,0 +1,20 @@
+parameters:
+  - $ref: '#/components/parameters/resourceId'
+post:
+  tags:
+    - Tracking
+  summary: Triggers webhook resending attempt for specified webhook tracking.
+  responses:
+    204:
```

```suggestion
    201:
```

+ `Location` header to `tracking@webhooks@{id}`

---

I'm not sure do we really need this header. Mentioned webhook is used only to retry it and during this POST request new webhook record is created but we don't have these newly created identifier of webhook tracking

---

We should forward the client to changed resource.

---

@slavcodev, what do you think?

---

As for `Location` header I suggest to not to add it to this endpoint as no any other endpoint defines it and add this header to all endpoints in separate PR. Anyway status for this endpoint was changed and response body was declared, thanks for advice!

---

No way to get new resource location, webhhok resend is scheduled and no new entity created at the moment of response, just like with webhook trigger preview